include/boost/geometry/extensions/gis/io/wkb/detail/parser.hpp

```diff
@@ -277,17 +277,14 @@ struct polygon_parser
             {
                 interior_rings(polygon).resize(rings_parsed);
                 ring_type& ringN = interior_rings(polygon).back();
+                
```

Comment:
This is a comment to the already existing code in a line above this one. Rings may not define `resize()` or `back()`. Please use e.g. `bg::range::resize()` and `bg::range::back()`.